keycloak/keycloak-ui
# Added an inline Alert for Signed Jwt authenticator in Credentials #1850

## Motivation
Closes https://github.com/keycloak/keycloak-ui/issues/1850
Fixes as per designs https://marvelapp.com/prototype/695h94c/screen/72266137

Note: I had to introduce a custom CSS class because patternfly was only offering a max `p-u-ml-4xl` and I needed more margin spacing.

## Verification Steps
1. Go to `Clients >> Select Client >> Credentials tab >> Select Signed Jwt`.
2. Verify that the Alert is present.
3. Click on `Save` btn and verify that the Alert is removed.

## Checklist:
